alvr/server/build.rs

```diff
@@ -79,21 +83,58 @@ fn main() {
     // #[cfg(debug_assertions)]
     // build.define("ALVR_DEBUG_LOG", None);
 
-    let use_ffmpeg = cfg!(feature = "gpl") || cfg!(target_os = "linux");
+    let gpl_or_linux = cfg!(feature = "gpl") || cfg!(target_os = "linux");
 
-    if use_ffmpeg {
+    if gpl_or_linux {
         let ffmpeg_path = get_ffmpeg_path();
 
         assert!(ffmpeg_path.join("include").exists());
         build.include(ffmpeg_path.join("include"));
     }
 
+    #[cfg(target_os = "linux")]
+    {
+        let x264_path = get_linux_x264_path();
+
+        assert!(x264_path.join("include").exists());
+        build.include(x264_path.join("include"));
+    }
+
     #[cfg(feature = "gpl")]
     build.define("ALVR_GPL", None);
 
     build.compile("bindings");
 
-    if use_ffmpeg {
+    #[cfg(target_os = "linux")]
```

Comment:
should we check for gpl here?

Reply:
For linux we don't check for GPL because linux without FFmpeg would be too crippled. And until now we always used FFmpeg with x264, that's why the license is GPL.

Reply:
I think so